cookbooks/omnitruck/templates/config.toml.erb

```diff
@@ -0,0 +1,5 @@
+[cfg.services.chef-es.omnitruck.delivered]
+start = "--permanent-peer"
+
+[cfg.services.chef-es.omnitruck-unicorn-proxy.delivered]
+start = "--permanent-peer"
```

Comment:
I'll remove this. It was for the director configuration. However, the director is removed upstream from Habitat, and that functionality will be baked into the supervisor, we'll use the `--listen` options